cuda_bindings/tests/test_nvjitlink.py

```diff
@@ -53,7 +55,7 @@ def test_invalid_arch_error():
 
 
 def test_create_and_destroy():
-    handle = nvjitlink.create(1, ["-arch=sm_53"])
+    handle = nvjitlink.create(1, ["-arch=sm_90"])

```

Comment:
btw if we are using `sm_90` everywhere, can we make it a global variable, e.g. `TEST_ARCH = "sm_90"` for the entire test module (with a comment explaining why locking at sm90 is OK here)?

Reply:
I changed the file to target a few different combinations and define the required globals at the top of the file